tools/inspector_protocol/code_generator.py

```diff
@@ -681,7 +681,7 @@ def generate_lib_file(file_name, template_files):
         generate_lib_file(os.path.join(config.lib.output, to_file_name(config, "base_string_adapter.h")), base_string_adapter_h_templates)
         generate_lib_file(os.path.join(config.lib.output, to_file_name(config, "base_string_adapter.cc")), base_string_adapter_cc_templates)
 
-    # Make gyp / make generatos happy, otherwise make rebuilds world.
```

Comment:
Can you revert all `inspector_protocol` changes, they should be upstreamed to <https://chromium.googlesource.com/deps/inspector_protocol/>